frontend/src/scenes/billing/Billing.tsx

```diff
@@ -138,16 +139,24 @@ export function Billing(): JSX.Element {
             )}
 
             {billing?.trial ? (
-                <LemonBanner type="info" className="mb-2">
-                    You are currently on a free trial for <b>{toSentenceCase(billing.trial.target)} plan</b> until{' '}
-                    <b>{dayjs(billing.trial.expires_at).format('LL')}</b>. At the end of the trial{' '}
-                    {billing.trial.type === 'autosubscribe'
-                        ? 'you will be automatically subscribed to the plan.'
-                        : 'you will be asked to subscribe. If you choose not to, you will lose access to the features.'}
+                <LemonBanner type="info" hideIcon className="mb-2">
+                    <div className="flex items-center gap-4">
+                        {billing.trial.type === 'autosubscribe' ? (
+                            <StarHog className="w-20 h-20 flex-shrink-0" />
+                        ) : (
+                            <BuilderHog3 className="w-20 h-20 flex-shrink-0" />
+                        )}
+                        <div>
```

Comment:
Hedgehogs are great, looks a little odd with spacing, maybe just add a title like "... and you get a PostHog trial" or something weird like that

Reply:
Oh maybe a judge hedgehog (new one) with headline "... and you're on trial"?

Reply:
Ha that's funny, yeah feel free to request art work for that and we can ship with another hog in it's place until it's ready. 

Maybe `You're on [a] trial` just to make it clear, but still funny.